README.md

```diff
@@ -270,77 +270,28 @@ For testing the API keys, please refer to this [documentation](./docs/api_key.md
 
 ### Run
 
-There are multiple ways in which we can now use the neuro-san server with a client:
+Neuro SAN Studio provides a user-friendly environment to interact with agent networks.
```

Comment:
Simplified the `Run` section to keep only the `nsflow` option by default. Note that CLI and the Flask web client are still supported but at that stage in the README they add unnecessary complexity, and I don't believe they were widely used.
`neuro-san-web-client` moved down to a 'Community Projects` section.